addons/ai/initSettings.sqf

```diff
@@ -0,0 +1,19 @@
+private _category = format ["ACE %1", LLSTRING(DisplayName)];
+
+[
+    QGVAR(assignNVG), "CHECKBOX",
+    [LSTRING(AssignNVG_DisplayName), LSTRING(AssignNVG_Description)],
+    _category,
+    false,
+    1,
+    {
+        if (isServer) then {
+            params ["_enabled"];
+            if (_enabled) then {
```

Comment:
Personally I would add a check that prevents 2 PFHs running at the same time and trying to remove a non-existent PFH.
I'm not sure how necessary those checks would be though.

Reply:
Doesn't hurt indeed.